RELEASENOTES.md

```diff
@@ -1,2 +1,3 @@
 - `integrate-boards` no longer requires a PAT with the `All Accessible Organizations` setting
-- Fixed incorrect repo url in migration logs when migrating from GHES
\ No newline at end of file
+- Fixed incorrect repo url in migration logs when migrating from GHES
+- Add `generate-mannequin-csv` command to generate a CSV to be used to reclaim mannequins in bulk.
```

Comment:
lets leave this out of the release notes until we remove the IsHidden flag

Reply:
done